requirements.txt

```diff
@@ -0,0 +1,186 @@
+# This file was autogenerated by uv via the following command:
```

Comment:
can you automate generating this file in the development pipeline somewhere in git hook?
if difficult, modify README to make sure updating this file when modifying dependencies